### DISCUSSION TITLE
macOS出现文件损坏问题

### DISCUSSION THREAD

**Discussion by JasondeWest** [PeiPei233/zju-learning-assistant#11] [Ideas]
打开终端输入这串代码：
sudo xattr -r -d com.apple.quarantine /Applications/zju-learning-assistant.app
然后输入密码即可

**Comment by PeiPei233** [comment]
> 打开终端输入这串代码：
> sudo xattr -d com.apple.quarantine /Applications/zju-learning-assistant.app
> 然后输入密码即可

原来是这样😯 破案了 为啥Apple这种显示的是损坏😇
